<launch>
    <node name="joy" pkg="joy" type="joy_node" >
        <param name="autorepeat_rate" value="30" />
        </node>
    <node name="joy_throttled" type="throttle" pkg="topic_tools" args="messages /joy 30 /joy_throttled" />
    <node name="inverse_kinematics" pkg="inverse_kinematics" type="inverse_kinematics_solver.py" cwd="node"/>
    <node name="biped" pkg="inverse_kinematics" type="activate_servo.py" output="screen" cwd="node"/>
    <node pkg="rostopic" type="rostopic" name="rostopic_echo_node" output="screen" args="echo /joint_angles" />
</launch>
